CelebrityStalker.xcodeproj/project.pbxproj

```diff
@@ -67,6 +69,7 @@
 		3F0ACCC51BD5664400E3BDE1 /* CelebrityStalker */ = {
 			isa = PBXGroup;
 			children = (
+				A1C9FAA11BD9369300161C68 /* celebBio.JSON */,
 				3F0ACCDB1BD56B7C00E3BDE1 /* Controller */,
```

Comment:
*Another reference to the __JSON file__ here, adding it to the __Source Tree_ of the project.*